ql/src/test/org/apache/hadoop/hive/ql/lockmgr/TestDbTxnManager2.java

```diff
@@ -2324,142 +2315,6 @@ private void testConcurrentMergeInsertNoDuplicates(String query, boolean sharedW
     List res = new ArrayList();
     driver.getFetchTask().fetch(res);
     Assert.assertEquals("Duplicate records found", 4, res.size());
-    dropTable(new String[]{"target", "source"});
-  }
-
-  /**
-   * ValidTxnManager.isValidTxnListState can invalidate a snapshot if a relevant write transaction was committed
-   * between a query compilation and lock acquisition. When this happens we have to recompile the given query,
-   * otherwise we can miss reading partitions created between. The following three cases test these scenarios.
-   * @throws Exception ex
-   */
-  @Test
-  public void testMergeInsertDynamicPartitioningSequential() throws Exception {
```

Comment:
These tests should be added back in your next change, when we recompile without the reexec driver, to verify the dyn partitioning use-cases

Reply:
Ack.